packages/@aws-cdk/aws-codepipeline-actions/lib/ecs/deploy-action.ts

```diff
@@ -58,7 +67,13 @@ export class EcsDeployAction extends Action {
       resource: props.service,
     });
 
+    const deploymentTimeout: number | undefined = props.deploymentTimeout && props.deploymentTimeout.toMinutes({ integral: true });
+    if (typeof deploymentTimeout !== 'undefined' && (deploymentTimeout < 1 || deploymentTimeout > 60)) {
+      throw new Error('Deployment timeout has to be between 1 and 60 minutes');
```

Comment:
Can you add the value that was provided here?

```suggestion
      throw new Error(`Deployment timeout must be between 1 and 60 minutes, got: ${deploymentTimeout}`);
```